test/EmailAddressTest.php

```diff
@@ -741,8 +741,8 @@ public function testUseMxCheckBasicValid()
         ];
 
         if (extension_loaded('intl')) {
-            $emailAddresses[] = 'иван@письмо.рф';
```

Comment:
So this is a breaking change, as you said an option should be implemented for choose what rules should be used